backend/api/Controllers/LookupController.cs

```diff
@@ -90,29 +91,31 @@ public IActionResult GetPropertyClassifications()
         /// <returns></returns>
         [HttpGet("all")]
         [Produces("application/json")]
-        [ProducesResponseType(typeof(IEnumerable<Model.CodeModel>), 200)]
+        [ProducesResponseType(typeof(IEnumerable<Model.CodeModel<object>>), 200)]
```

Comment:
Quite a few changes here to support generics.  I'm not sure if this will cause issues in the frontend.  Primarily it means that the `id` property will be of different types based on the object.  I may need to convert it back to a string through the mapping if it causes issues.